tests/plugins/dialog/plugin.js

```diff
@@ -26,9 +50,19 @@
 		CKEDITOR.dialog.add( 'testDialog2', '%TEST_DIR%_assets/testdialog.js' );
 	} );
 
-	bender.editor = {};
+	bender.editor = {
+		config: {
+			dialog_defaultValues: {
+				'testDialog1.info.text1': 'text1',
+				'testDialog1.info.text2': 'text2'
+			}
+		}
+	};
 
 	bender.test( {
+		setUp: function() {
+			this.editor.addCommand( 'testDialog1', new CKEDITOR.dialogCommand( 'testDialog1' ) );
```

Comment:
Technically this is invalid construction. You're not supposed to add the same command multiple times, and this is what is happening here. `setUp` runs for each TC.

Reply:
Instead you could simply do that at the end of `instanceLoaded` event.